app/helpers/sessions_helper.rb

```diff
@@ -4,7 +4,7 @@ def log_in user
     session[:user_id] = user.id
   end
 
-  def remember(user)
+  def remember user
     user.remember
     cookies.permanent.signed[:user_id] = user.id
```

Comment:
What is signed cookie? What is the purpose of using it?

Reply:
cookies.signed use with variable can security
Set-Cookie: demo_signed=InNpZ25lZCI%3D--365ab5f6
? Key InNpZ25lZCI%3D--365ab5f6 save config/initializers/secret_token.rb:
Purpose:
Data cookie save as encode when I create cookies